cl/alerts/views.py

```diff
@@ -108,10 +108,16 @@ def toggle_docket_alert(request: AuthenticatedHttpRequest) -> HttpResponse:
         )
         if existing_alert.exists():
             if existing_alert[0].alert_type == DocketAlert.SUBSCRIPTION:
-                existing_alert.update(alert_type=DocketAlert.UNSUBSCRIPTION)
+                # Use save() to force date_created to be updated
+                da_alert = existing_alert[0]
+                da_alert.alert_type = DocketAlert.UNSUBSCRIPTION
+                da_alert.save()
```

Comment:
This kind of a hack to explicitly use the save method and implicitly break the update method ain't great.

Reply:
Well, now we use the `date_modified` (that uses `auto_now`) to keep track of old docket alerts, so we need to use the `save()` method here so `date_modified` is updated when docket alert is toggled.